spk/rutorrent/src/.rtorrent.rc

```diff
@@ -5,6 +5,6 @@ schedule = watch_directory,5,5,load_start=@watch_dir@/*.torrent
 max_memory_usage = @max_memory@
 log.open_file = "rtorrent.log", "/usr/local/rutorrent/var/rtorrent.log"
 log.add_output = "warn", "rtorrent.log"
-http_cacert = /etc/ssl/certs/ca-certificates.crt
+network.http.cacert = /etc/ssl/certs/ca-certificates.crt
```

Comment:
Since `rtorrent` upgrade this is the new parameter name.

Note: this means that current rutorrent ui won't be able to change that value (it may even break the configuration to be honest but I didn't take the time to check this)